dev-helpers/dev-helper-initializer.js

```diff
@@ -4,7 +4,9 @@ window.onload = function() {
   window["SwaggerUIStandalonePreset"] = window["swagger-ui-standalone-preset"]
   // Build a system
   const ui = SwaggerUIBundle({
-    url: "https://petstore.swagger.io/v2/swagger.json",
+    urls: [
+      {url:"https://petstore3.swagger.io/api/v3/openapi.json",name:"Pet Store Example", servers:[{ "url": "http://test.com" },{ "url": "http://real.org" }]}, 
```

Comment:
can we pass the servers list from configmap or ENV var?

Reply:
Yes, 
They support pass URLs as Docker ENV
https://swagger.io/docs/open-source-tools/swagger-ui/usage/configuration/#:~:text=urls-,URLS,-Array.%20An%20array